AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/AWSDataStorePlugin+DataStoreBaseBehavior.swift

```diff
@@ -355,6 +354,11 @@ extension AWSDataStorePlugin: DataStoreBaseBehavior {
         guard #available(iOS 13.0, *) else {
             return
         }
+
+        guard let storageEngine = storageEngine else {
+            return
```

Comment:
Lets add a verbose log here so that we can debug if something goes wrong.

Reply:
will add the log